tests/unit/components/async-button-test.js

```diff
@@ -1,90 +0,0 @@
-import {
-  moduleForComponent,
```

Comment:
Converted to integration tests. Unit tests don't work without jQuery, but regardless of I find them rather useless.